devtools/src/main.rs

```diff
@@ -0,0 +1,25 @@
+extern crate actix;
+extern crate actix_web;
+extern crate structopt;
+
+use structopt::StructOpt;
+
+mod server;
+
+#[derive(StructOpt, Debug)]
+#[structopt(name = "dev")]
+struct Opt {
+    #[structopt(short, long, default_value = "8080")]
+    port: u32,
+    #[structopt(short, long, default_value = "localhost")]
+    host: String,
+}
```

Comment:
maybe would be great to have the opposability to  set port and host by .env file. For that you will need https://docs.rs/dotenv/0.15.0/dotenv/fn.dotenv.html

Reply:
I'll add support for this (thanks)!